src/lib/titled-hoc.jsx

```diff
@@ -15,34 +13,24 @@ const TitledHOC = function (WrappedComponent) {
                 'handleUpdateProjectTitle'
             ]);
             this.state = {
-                projectTitle: this.props.intl.formatMessage(defaultProjectTitleMessages.defaultProjectTitle)
+                projectTitle: ''
```

Comment:
I think this should be 'Scratch Project'. (Since I was suggesting we don't set the project-title-input to defaultProjectTitleMessages.defaultProjectTitle

Reply:
The issue with hard coding 'Scratch Project' is that it will never get translated when the GUI is stand-alone, unless `setReduxTitle` in gui also checks for `newTitle === 'Scratch Project`. If that seems acceptable, then this change makes sense.

Reply:
The better solution is to default projectTitle to null, then the title handling in GUI already handles it properly without having to hard code `'Scratch Project'`.

Reply:
Chris came up with a better way, making this `null` and relying on `intl`-injected GUI to detect that and set it.